api/processor_api.py

```diff
@@ -393,12 +513,12 @@ async def replace_camera_image(camera_id, body: ImageModel):
             except Exception:
                 return HTTPException(status_code=400, detail="Invalid image format")
 
-        @app.post("/{camera_id}/homography_matrix")
+        @app.post("cameras/{camera_id}/homography_matrix")
         async def config_calibrated_distance(camera_id, body: ConfigHomographyMatrix):
             sources = self.config.get_video_sources()
             dir_source = list(filter(lambda source: source['id'] == camera_id, sources))
             if dir_source is None or len(dir_source) != 1:
-                raise HTTPException(status_code=404, detail=f'Camera with id "{camera_id}" does not exist')
+                raise HTTPException(status_code=404, detail=f'The camera: {camera_id} does not exist')
             dir_source = dir_source[0]
```

Comment:
Maybe we can avoid the line `dir_source = dir_source[0]` using the operator `next` in the line 519 (as @mats-claassen suggested)